resource/chatadapter_test.go

```diff
@@ -0,0 +1,54 @@
+package resource_test
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Pix4D/cogito/resource"
+)
+
+func TestChatAdapterMock(t *testing.T) {
```

Comment:
Can you explain what this is? and why its empty?

Reply:
Ah, I had forgotten about this :-/